typed-racket-more/typed/images/logos.rkt

```diff
@@ -0,0 +1,22 @@
+#lang typed/racket
+
+(require typed/racket/draw)
+
+(module digitama typed/racket
+  (provide (all-defined-out))
+  
+  (define-syntax (require/typed/provide/batch stx)
```

Comment:
I think this should go in `typed/private`, since it's probably useful elsewhere.